api/api/admin/media_report.py

```diff
@@ -866,10 +890,6 @@ def is_pending(self, obj):
         "is_pending",
         "media_id",  # used because ``media_obj`` does not render a link
     )
-    list_filter = (
```

Comment:
Why is this filter removed, @krysal ?

Reply:
We need to reduce potential issues when rendering the page, and the filters declared this way will result in more `DISTINCT` queries, which are prohibitively expensive in the image table. After confirming we can access the image list, we can re-add these filters optimally (like done here with sources), although I don't see them listed in the admin currently.